kits/fashion-outfit-analyzer/agent.md

```
@@ -0,0 +1,24 @@
+# Fashion Outfit Analyzer
+
+## Agent Overview
```

markdownlint-cli2 (0.22.1)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)